api/app/settings/common.py

```diff
@@ -217,6 +217,7 @@
 
 LOGIN_THROTTLE_RATE = env("LOGIN_THROTTLE_RATE", "20/min")
 SIGNUP_THROTTLE_RATE = env("SIGNUP_THROTTLE_RATE", "10000/min")
+USER_THROTTLE_RATE = env("USER_THROTTLE_RATE", "10/second")
```

Comment:
@matthewelwell we need to decide a sensible limit here;

Reply:
Hmm... yeah, good question. 10/second seems reasonable to me for now tbh.

Reply:
Maybe 500/min is better?

Reply:
Yeah, think we should start with a higher limit